ci/job.yml

```diff
@@ -166,7 +166,7 @@
             read CICO_hostname CICO_ssid <<< $(cico node get -f value -c ip_address -c comment)
             sshopts="-t -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -l root"
             ssh_cmd="ssh $sshopts $CICO_hostname"
-            $ssh_cmd yum -y install rsync git PyYAML
+            $ssh_cmd "yum -y install epel-release && yum -y install rsync git PyYAML python-networkx"
```

Comment:
In the future, we could move this inline script to a different file. This will remove the need to update the jobs when the script is updated.